modules/admin/views/settings_viewer.py

```diff
@@ -0,0 +1,40 @@
+import ez
+
+from pydantic.fields import FieldInfo
+
+from ez.plugins import Settings
+
+from seamless.html import Div
+from seamless.styling import Style
+
+
+def render_settings_viewer(settings: Settings):
+    def render_section(section: type[Settings], value: Settings):
+        return Div()(
+            Div()(section.__ez_section_title__),
+            render_settings_viewer(value)
+        )
+
+    def render_field(name: str, field: FieldInfo, value):
+        annotation = field.annotation
+        if not isinstance(annotation, type):
+            raise TypeError
+        provider_type = ez.data.providers.get_provider_type(annotation)
+        provider = provider_type.load(value)
+        return Div(style=Style(
```

Comment:
This is bad practice but I’ll approve it for now, we just need to make sure it’s changed before release.

Reply:
I never intended to create the UI myself. This was just a proof of concept and it gave my an idea of how to implement plugin settings.

Reply:
Aren’t you the frentend developer?